_posts/2016/2016-04-20-jenadevs-docker-party.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "Jena Developers shipped the Docker Party at ePages!"
+date: "2016-04-20 12:04:20"
+icon: cubes
+categories: events
+authors: ["Benjamin and Oliver Z."]
+---
+
+On the first Thursday of April the newly founded [Softwerkskammer Jena](https://www.softwerkskammer.org/groups/jena) had its first [Developers Meetup](www.meetup.com/jenadevs).
```

Comment:
I would not specify the day, as we already have mid/end April.
I'd rather go for "Beginning of April..." or completely leave that out and just say " The Softwerkskammer Jena had its first..."
But still, this sounds not like the ePages perspective.